easybuild/easyconfigs/e/Elmer/Elmer-9.0-DCRComplexSolve.patch

```diff
@@ -0,0 +1,428 @@
+Backport to 9.0 to allow compile with foss/2022b compiler.
```

Comment:
Since you say 'backport', I'm assuming this patch is based on some upstream PR, maybe related to an upstream issue... if so: could you link those in the comments above the patch? It's always good to have references to upstream for context. It helps future contributors to e.g. figure out if this patch is still needed.

Reply:
Updated all backport patches with PR references.